tools/antler-run/file-utils.hpp

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <eosio/vm/constants.hpp>
+
+#include <fstream>
+#include <cassert>
+#include <vector>
+#include <string_view>
+#include <cstring>
+
+namespace eosio { namespace utils {
+
+enum class file_type {
+    uninitialized,
+    non_elf_other,
+    elf_object,
+    elf_executable,
+    elf_shared_object,
+    elf_core_dump,
+    wasm
+};
+
+file_type get_file_type(const char* path) {
```

Comment:
make this `inline`.

Reply:
will do.
p.s. not a big deal anyway but internet suggests:
`The compiler declines the request of inlining a function if it contains any switch or go-to statements`

Reply:
Resolved.